test/integration/addListing.js

```diff
@@ -272,210 +291,135 @@ module.exports = async (artifacts, web3) => {
-    const [_, applicant, challenger, voterFor, voterAgainst, buyer1, buyer2] = accounts
-    const applicantBalance = (await token.balanceOf.call(applicant)).toNumber()/10**18
-    const challengerBalance = (await token.balanceOf.call(challenger)).toNumber()/10**18
-    const voterForBalance = (await token.balanceOf.call(voterFor)).toNumber()/10**18
-    const voterAgainstBalance = (await token.balanceOf.call(voterAgainst)).toNumber()/10**18
-    const registryBalance = (await token.balanceOf.call(registry.address)).toNumber()/10**18
-    const buyer1Balance = (await token.balanceOf.call(buyer1)).toNumber()/10**18
-    const buyer2Balance = (await token.balanceOf.call(buyer2)).toNumber()/10**18
-    console.log('')
-    console.log('')
-    console.log('')
-    console.log('balances:')
-    console.log(`  applicant:  ${applicantBalance}`)
-    console.log(`  challenger: ${challengerBalance}`)
-    console.log(`  buyer1:     ${buyer1Balance}`)
-    console.log(`  buyer2:     ${buyer2Balance}`)
-    console.log(`  Registry Contract: ${registryBalance}`)
-    if(challenge) {
-      const challengeBalance = (await token.balanceOf.call(challenge.address)).toNumber()/10**18
-      console.log(`  Challenge Contract: ${challengeBalance}`)
-    } else {
-      console.log('  Challenge Contract: NULL')
-    }
-    if(futarchyOracle) {
-      const futarchyBalance = (await token.balanceOf.call(futarchyOracle.address)).toNumber()/10**18
-      console.log(`  Futarchy Oracle Contract: ${futarchyBalance}`)
-    } else {
-      console.log('  Futarchy Oracle Contract: NULL')
-    }
-    if(catEvent) {
-      const catEventBalance = (await token.balanceOf.call(catEvent.address)).toNumber()/10**18
-      console.log(`  Categorical Event: ${catEventBalance}`)
-    } else {
-      console.log('   Categorical Event: NULL')
-    }
-    if(aScal) {
-      const acceptedToken = await OutcomeToken.at(await aScal.collateralToken())
-      const aScalBalance = (await acceptedToken.balanceOf.call(aScal.address)).toNumber()/10**18
-      console.log(`  Scalar Accepted Event: ${aScalBalance}`)
-    } else {
-      console.log('   Scalar Accepted Event: NULL')
-    }
-    if(dScal) {
-      const acceptedToken = await OutcomeToken.at(await dScal.collateralToken())
-      const dScalBalance = (await acceptedToken.balanceOf.call(dScal.address)).toNumber()/10**18
-      console.log(`  Denied Accepted Event: ${dScalBalance}`)
-    } else {
-      console.log('   Denied Accepted Event: NULL')
-    }
-    console.log('')
-    console.log('')
-    console.log('')
-  }
-  
-  async function logRegistryStatus(registry) {
-    console.log('----------')
-    console.log('REGISTRY STATUS')
-    console.log('----------')
-  }
-  
-  async function getFutarchyChallenge(challengeID, registry) {
-    const challenge = (await registry.challenges(challengeID))[0]
-    return FutarchyChallenge.at(challenge)
-  }
+  // async function marketBuy (market, outcomeTokenIndex, arrayGuy, from) {
```

Comment:
wondering why you want to leave commented lines

Reply:
removed these